alert/alert.go

```diff
@@ -52,6 +52,9 @@ type Alert interface {
 	// GetKvdbInstance.
 	GetKvdbInstance() kvdb.Kvdb
 
+	// RaiseSingleton raises a singleton alert.
+	RaiseSingleton(alert *api.Alert) error
```

Comment:
This doesn't make sense to me. What does RaiseSingleton mean? To me it sounds like a single object is being.. raised? Not sure.